source/tyr/tyr/command/import_last_autocomplete_dataset.py

```diff
@@ -56,12 +56,38 @@ def import_last_autocomplete_dataset(instance_name, wait=True):
 
 
 @manager.command
+def import_last_stop_dataset(instance_name, wait=True):
+    """
+    reimport the last datasets of of all instances with import_stops_in_mimir = true
+
+    you can set wait=False if you don't want to wait for the result
+    """
+    instance = models.Instance.query_existing().filter_by(name=instance_name).first()
+
+    if not instance:
+        raise Exception("cannot find instance {}".format(instance_name))
+
+    files = [d.name for d in instance.last_datasets(1)]
+    logger = logging.getLogger(__name__)
+    logger.info('we reimport to mimir the last dataset of %s, composed of: %s',
+                instance.name, files)
+    future = tasks.import_in_mimir(files, instance, async=wait)
```

Comment:
this won't work, you'll send some osm/synonym data too mimir too, and the function `import_in_mimir` does not check the type of data